client/blocks/comment-button/index.jsx

```diff
@@ -56,14 +56,14 @@ class CommentButton extends Component {
 					<span className="comment-button__label-count">{ commentCount }</span>
 				) }
 				{ showLabel &&
-				commentCount > 0 && (
-					<span className="comment-button__label-status">
-						{ translate( 'Comment', 'Comments', {
-							context: 'noun',
-							count: commentCount,
-						} ) }
-					</span>
-				) }
+					commentCount > 0 && (
+						<span className="comment-button__label-status">
+							{ translate( 'Comment', 'Comments', {
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.